lib/core/res/colors.dart

```diff
@@ -37,6 +37,11 @@ class C {
   /// button colors
   static final Color authButtonColor = HexColor("#A955F7");
   static final Color speakerButtonColor = HexColor('#8E31D5');
+
+  /// FrameCardWidget colors
+  static final Color downArrowColor = HexColor("#FD8020");
+  static final Color blendSocialIconColorOne = Colors.black12;
+  static final Color blendSocialIconColorTwo = Colors.black38;
```

Comment:
use `static const` for pre-defined colors,
```suggestion
  static const Color blendSocialIconColorOne = Colors.black12;
  static const Color blendSocialIconColorTwo = Colors.black38;
```